### PR TITLE
Create README.md for our repository #25

### LINKED ISSUE #25: To create a stunning README.md for our repository
Problem: Our current README.md is plain & has little details. We need a more attractive README.md document.

Solution: Use your creative juices to spruce up our README.md documentation that attract users to join our open source project!

Expectation: To create a stunning README.md with **proper markdown formatting** that attracts users to join & contribute to our open-source project! **Include images as well - e.g. such as the Hacktoberfest 2022 logo/banner**. You can refer to other repositories for the general format.

Take note: You can take reference from the README.md of other repositories. But **DO NOT PLAGIARIZE** or else your PR would be labeled as spam & disqualified from Hacktoberfest 2022. Take this time to understand how others have written their README.md documentation & try to write your own unique README.md which you are proud of showing to the rest of our community.

**This is a very beginner-friendly task that allows you to practice your Markdown skills, and contribute to a meaningful cause in a low to no-code manner!**

### PR DESCRIPTION
**Pull request by Bennykillua**
This resolve #25

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,74 @@
...
+   * Type `git --version` in a directory of your choice - you should see the version of Git that you downloaded. If you see an error, close and reopen VS Code and try again.
+   * Set your user name with 
+   `git config --global user.name "<your-first-name> <your-last-name>"` - replace the `<placeholders>` with the real values
+   * Set your email address with 
+   `git config --global user.email "<your-email-here>"`- replace the `<placeholder>` with the real values (your email address should match the one with which you signed up for your GitHub account)
+
+1.  Fork this repository. This will create a copy of this repo in your GitHub account:
+
+    * Open [Your-First-Open-Source-Project/Hacktoberfest2022](https://github.com/Your-First-Open-Source-Project/Hacktoberfest2022) (this repository)
+    * Select **Fork** and  * Make a [Fork](https://github.com/Your-First-Open-Source-Project/Hacktoberfest2022/fork) --> The URL of your fork is now `https://github.com/<YOUR GITHUB ACCOUNT>/blog/`
```
Comment (melvincwng) [inline README.md:52]: Line 52 - I believe URL of fork isn't `https://github.com/<YOUR GITHUB ACCOUNT>/blog/`

It should be `https://github.com/<YOUR GITHUB ACCOUNT>/Hacktoberfest2022`

File: README.md [hunk 2]
```diff
@@ -1,2 +1,74 @@
...
+1. Clone the repository
+2. Make chnages 
+3. Commit and push your changes to your fork
+* type `git add .` (yes, there is a space (` `) between `add` and the `.`) - this adds all changes to staging area
+* type `git commit -m "YOUR COMMIT MESSAGE"` - this will commit your changes with the messages
+* type `git push` to push the changes to your remote fork
+
+9. make a Pull request
+
+## what kind of contribution can I make? ⭐
```
Comment (melvincwng) [inline README.md:63]: Change 'what' to 'What' for consistency purposes

File: README.md [hunk 3]
```diff
@@ -1,2 +1,74 @@
...
+   * Set your email address with 
+   `git config --global user.email "<your-email-here>"`- replace the `<placeholder>` with the real values (your email address should match the one with which you signed up for your GitHub account)
+
+1.  Fork this repository. This will create a copy of this repo in your GitHub account:
+
+    * Open [Your-First-Open-Source-Project/Hacktoberfest2022](https://github.com/Your-First-Open-Source-Project/Hacktoberfest2022) (this repository)
+    * Select **Fork** and  * Make a [Fork](https://github.com/Your-First-Open-Source-Project/Hacktoberfest2022/fork) --> The URL of your fork is now `https://github.com/<YOUR GITHUB ACCOUNT>/blog/`
+
+1. Clone the repository
+2. Make chnages
```
Comment (melvincwng) [inline README.md:55]: There is a typo on the word 'changes'. Pls fix it.

**Review by melvincwng — CHANGES_REQUESTED** [review]
Awesowe work @Bennykillua, there are 3 minor issues with the PR which I need you to fix before I merge the PR.

Pls take a look at the comments I've listed and edit the PR accordingly.

Looking forward to your revised PR, thanks

**Comment by melvincwng** [comment]
@Bennykillua merging the PR first

Will assign others to do the minor fixes later

**Review by melvincwng — APPROVED** [review]

**Comment by Bennykillua** [comment]
I will do the fixes @melvincwng

**Comment by Bennykillua** [comment]
You have already made the fixes. Thanks @melvincwng

**Pull request merged**